src/screens/OavMover.tsx

```diff
@@ -175,6 +191,73 @@ export function PixelsToMicrons({
   );
 }
 
+export function PresetMovements() {
+  return (
+    <Box sx={{ textAlign: `center` }}>
+      <p>
+        <b>Preset Positions</b>
+      </p>
+      <Grid2>
+        <Tooltip title={"Move into position for collection"}>
+          <Button
+            style={buttonStyle}
+            onClick={() =>
+              submitAndRunPlanImmediately("moveto_preset", {
+                place: "collect_position",
+              })
+            }
+          >
+            Collect Position
+          </Button>
+        </Tooltip>
+
+        <Tooltip title={"Move hardware for sample loading"}>
+          <Button
+            style={buttonStyle}
+            onClick={() =>
+              submitAndRunPlanImmediately("moveto_preset", {
+                place: "load_position",
+              })
+            }
+          >
+            Load Position
+          </Button>
+        </Tooltip>
+        <Tooltip title={"Align microdrop"}>
+          <Button
+            style={buttonStyle}
+            onClick={() =>
+              submitAndRunPlanImmediately("moveto_preset", {
+                place: "microdrop_position",
+              })
+            }
+          >
+            Microdrop Align
+          </Button>
+        </Tooltip>
+      </Grid2>
+    </Box>
+  );
+}
+
+export function SideDrawer() {
+  const [open, setOpen] = useState(false);
+  const toggleDrawer = (newOpen: boolean) => () => {
+    setOpen(newOpen);
+  };
+
+  return (
+    <>
+      <Button style={buttonStyle} onClick={toggleDrawer(true)}>
+        Open drawer
```

Comment:
This button should say something recognisable to the user, "drawer" is a bit too generic - for the moment I think we can keep the "Preset Positions" wording from the edms...